src/resources/i18n/translations/en.json

```diff
@@ -1,378 +1,378 @@
 {
-	"18 per year after that": "$18 per year after that",
+	"18 per year after that": "$18 yearly afterwards",
 	"A-Z": "A-Z",
-	"ARIA HINT - Contact button": "open your email client and email us.",
+	"ARIA HINT - Contact button": "open your e-mail program and send us an e-mail.",
 	"ARIA HINT - Current playback time": "Current playback time",
 	"ARIA HINT - Currently selected filter": "Currently selected filter",
 	"ARIA HINT - Error something went wrong with playing this item": "Could not play this.",
 	"ARIA HINT - Hide podcast settings": "Hide podcast settings",
-	"ARIA HINT - Max playback speed": "Set the highest available playback speed for your player",
-	"ARIA HINT - Membership - Create and share clips": "Create and share clips in Podverse premium.",
-	"ARIA HINT - Membership - Create playlists": "Create playlists in Podverse premium.",
-	"ARIA HINT - Membership - Download a backup of your data": "Download a backup of your data in Podverse premium.",
+	"ARIA HINT - Max playback speed": "Set max playback speed for your player",
+	"ARIA HINT - Membership - Create and share clips": "Create and share clips with Podverse premium.",
+	"ARIA HINT - Membership - Create playlists": "Create playlists with Podverse premium.",
+	"ARIA HINT - Membership - Download a backup of your data": "Download a backup of your data with Podverse premium.",
 	"ARIA HINT - Membership - Download episodes": "Download episodes.",
 	"ARIA HINT - Membership - Drag and drop queue": "Player queue.",
 	"ARIA HINT - Membership - Sleep timer": "Sleep timer.",
 	"ARIA HINT - Membership - Subscribe to podcasts": "Subscribe to podcasts.",
-	"ARIA HINT - Membership - Support free and open source software": "Support free and open source software 😊",
-	"ARIA HINT - Membership - Sync your queue on all devices": "Sync your queue to all devices in Podverse premium.",
-	"ARIA HINT - Membership - Sync your subscriptions on all devices": "Sync your subscriptions to all devices in Podverse Premium.",
-	"ARIA HINT - Membership features header": "These are the features the app presented as free, and premium features.",
+	"ARIA HINT - Membership - Support free and open source software": "Support copylefted libre software 😊",
```

Comment:
Ambiguous use of "free" in Free Software confuses the aspect of freedom for freeware.
Similarly "the free period" is now called "gratis period of Podverse premium".